### ISSUE TITLE
Allow webpack configurations written in any language

### ISSUE THREAD

**Issue by themaxdavitt** [cloudflare/wrangler#1770]
#### Overview and problem statement

I like to write my [`webpack` configurations using TypeScript](https://webpack.js.org/configuration/configuration-languages/#typescript), but `wrangler` expects JavaScript files, so it throws errors, e.g.:
```
(node:2110764) UnhandledPromiseRejectionWarning: /home/user/wrangler_webpack_ts/webpack.config.ts:1
import * as path from 'path';
^^^^^^

SyntaxError: Cannot use import statement outside a module
...
```

I should be able to use `webpack` configuration files written in any language supported by `webpack`.

#### Basic example

I should be able to have `webpack_config = "webpack.config.ts"` in my `wrangler.toml` and, provided I have [`ts-node`](https://www.npmjs.com/package/ts-node) installed (or whatever is listed in [this](https://www.npmjs.com/package/interpret), which is what `webpack` uses for this functionality), it should just work.

#### Remarks

This really isn't a huge issue because, for instance, if you still have `ts-node` installed, you can just have your `webpack.config.js` file look like this:
```javascript
require('ts-node/register');
module.exports = require('./webpack.config.ts').default;
```
Just thought I might as well ask. Feel free to close/merge into similar `webpack` issues - I saw a few but they seem to be larger in scope and I don't know if this request would get implemented by those.

Thanks in advance!

**Comment by xtuc** [comment]
It's not possible right now but once https://github.com/cloudflare/wrangler/pull/1677 lands it should fix that

**Comment by rita3ko** [comment]
Closing in favor of #1677 .
